### ISSUE TITLE
CSP Blocks java script execution in Firefox

### ISSUE THREAD

**Issue by sahnib** [grafana/grafana#34339]
**What happened**:

Enabling CSP resulted in errors in Firefox browser. 

**What you expected to happen**:

No errors with java script blocked due to enabling CSP. 

**How to reproduce it (as minimally and precisely as possible)**:

1. Set option `content_security_policy` to true in defaults.ini, and launch the Grafana server. 
2. Create a CloudWatch data source.
3. Import a default CW dashboard, 
4. Open the Developer tools console in browser. 
5. Open the dashboard. 

The console will show errors like :

```
Content Security Policy: The page’s settings blocked the loading of a resource at inline (“script-src”). jquery.js:6347

Content Security Policy: The page’s settings blocked the loading of a resource at inline (“script-src”). angular.js:3332
```

**Environment**: 
- Grafana version: v7.5.5
- Data source type & version: CloudWatch (have not tested with other datasources) 
- OS Grafana is installed on: Linux (Ubuntu)
- User OS & Browser: Linux, Firefox
- Grafana plugins: None
- Others: N/A.

**Comment by aknuds1** [comment]
I tried to reproduce in Firefox 88.0.1 on OSX, Grafana 7.5.6, I don't get these errors at all with content security policy enabled (in conf/custom.ini though, conf/defaults.ini should be untouched).

**Comment by aknuds1** [comment]
Sorry, I missed the part of loading a dashboard with CW data source, I can reproduce then.

**Comment by aknuds1** [comment]
I found a workaround, although it relaxes the security a lot. Not sure there's any alternative though until we fix the root issue. Change the script-src part of content_security_policy_template in conf/custom.ini as follows: `script-src 'self' 'unsafe-eval' 'unsafe-inline';`. I've found that this works in FF for me.

I believe the root issue is #33323, i.e. that Webpack doesn't set the nonce attribute on injected JS bundles, which probably causes these inline scripts (loaded by JS bundles?) to be rejected by FF.
